index.js

```diff
@@ -63,6 +94,16 @@ var new_client = function(api_key, config) {
       cb(new Error("[LaunchDarkly] No user specified in toggle call"), default_val);
     }
 
+    if (this.stream && this.seeded) {
```

Comment:
It would also be useful here to detect whether streaming mode has failed and we need to fall back to polling.